hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/cache/CacheIterateMessageTask.java

```diff
@@ -77,6 +78,6 @@ public Object[] getParameters() {
 
     @Override
     public String getMethodName() {
-        return "iterator";
+        return SecurityInterceptorConstants.FETCH;
```

Comment:
Should not this be SecurityInterceptorConstants.ITERATE? 🤔

Reply:
They actually correspond to the hasNext and next methods of the iterator. I am changing it back to iterator as suggested.